python/edm4hep/datamodel.py

```diff
@@ -0,0 +1,14 @@
+import importlib
```

Comment:
Can the name of this file start with a `_` to indicate that it is "private"?

Reply:
I'd rather not, I don't think that convention is used in modern python code